addons/mail/static/tests/composer/composer.test.js

```diff
@@ -89,6 +89,22 @@ test("add an emoji", async () => {
     await contains(".o-mail-Composer-input", { value: "😤" });
 });
 
+test("auto generate emoji from text", async () => {
```

Comment:
I would rename to "emojis are auto-substituted from text"
- "substitution" is the conventional verb for the feature
- test names are easier to read as a sentence. Starting with verb makes the test too obscure (test above "add an emoji" is poor)